python/ray/tests/test_numba.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from numba import njit
+import pytest
+import numpy as np
+import unittest
+import ray
+
+
+@njit(fastmath=True)
+def centroid(x, y):
+    return ((x / x.sum()) * y).sum()
+
+
+# A Wrapper to call centroid func
+@ray.remote
+def centroid_wrapper(x, y):
+    return centroid(x, y)
+
+
+class NumbaTest(unittest.TestCase):
+    def setUp(self):
+        ray.init()
+
+    def tearDown(self):
+        ray.shutdown()
+
+    def assertEqualHelper(self, func, expected, *args):
+        assert ray.get(func.remote(*args)) == expected
+
+    def test_numba_njit(self):
+        x = np.random.random(10)
+        y = np.random.random(1)
+        self.assertEqualHelper(centroid_wrapper, centroid(x, y), x, y)
+
+    def test_numba_fail(self):
```

Comment:
I think we should remove this test, since this isn't exactly the ideal behavior (ideally it would work).